eox_nelp/course_experience/api/v1/routers.py

```diff
@@ -1,11 +1,18 @@
 """Routes configuration for course experience views."""
 from rest_framework import routers
 
-from eox_nelp.course_experience.api.v1 import views
+from eox_nelp.course_experience.api.v1 import public_views, views
```

Comment:
I understand the point. But here I have something to discuss:
![Screenshot from 2023-06-08 10-28-10](https://github.com/eduNEXT/eox-nelp/assets/51926076/653eac7e-c4c3-4b9c-b70f-a007d468ffc8)
![Screenshot from 2023-06-08 10-30-25](https://github.com/eduNEXT/eox-nelp/assets/51926076/0b565549-7128-4825-ba05-cd57081cdc0f)

Both have good reasons to use it, I split because the docstrings make the `views.py` file so large. What do you think?

Reply:
Actually, I like the base views first option but would be a refactor to the future, by the moment I am going to set the main views file because it makes sense the routes to import multiple files seems a good reason.

Reply:
Hey, I refactored but also, after adding the many lines of strings explaining the method. I had to add this.:
[7ad49af](https://github.com/eduNEXT/eox-nelp/pull/53/commits/7ad49af85bc023c7560cb979b48fd97bcafcef8f)

Reply:
no worries I prefer a big string with short code that makes easier to understand rather than big code and a poor explanation